cfgov/cfgov/wsgi.py

```diff
@@ -23,6 +29,17 @@ def initialize_new_relic():
         newrelic.agent.initialize(new_relic_config_file)
 
 
+envfile_path = os.path.join(this_dir, '../../.env'),
+environmentdotjson_path = os.path.join(this_dir, '../../environment.json')
+
+if os.path.exists(envfile_path):
+    dotenv.read_dotenv(envfile_path)
```

Comment:
Do we want to use `override=True` here (see [docs](https://github.com/jpadilla/django-dotenv#usage))? That would make the .env file override any existing variables already in the environment. I think we do want that behavior, and that's how environment.json works now.

Reply:
I'd say I'm *mildly* uncomfortable with that-- when I read about it, it seemed like an elegant way to recognize that if a variable has been set through some other mechanism, it reflects somebody's intent.

That said, I don't feel too strongly. I'll go ahead and make the change.

Reply:
I think I found the reason no to use override=True: if .env takes precedence, then we can't set environment variables in docker-compose.

Reply:
>I think I found the reason not to use override=True: if .env takes precedence, then we can't set environment variables in docker-compose.

docker-compose itself has an [`env_file`](https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option) attribute, and it supports the same `KEY=value` format as django-dotenv.  So, maybe the better option for compose, is to **not** load the `.env` file in-container, and let compose inject those envvars itself?

We'll have to do something similar in a production environment.  We don't want to bake in env-specific `.env` files at image build time, and we won't be able to map in a `.env` at runtime, so the app will have to support just having those envvars already there at container start.  I _think_ that's fine in the current logic, since it doesn't fail if neither `.env` nor `environment.json` are present.

Also, is the plan to update [`.env_SAMPLE`](https://github.com/cfpb/cfgov-refresh/pull/5133/files#diff-eb951766c3e0a8d12740b6236ded5a9f) to the `KEY=value` style syntax?  Or do we need it to be an actual shell script for other purposes?

Reply:
I've removed override=True, so variables set via Docker, Linux, Compose, or tox will always take precedence.